### DISCUSSION TITLE
Shield Design V2.3

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#171] [Shield Design]
<img width="2032" alt="PCB 3D View" src="https://github.com/DeeEmm/DIY-Flow-Bench/blob/master/PCB/PCB%203D.png?raw=true">

Shield Design V2.3 is now available.

This design addresses the issues found with the V2 shields. (https://github.com/DeeEmm/DIY-Flow-Bench/discussions/129)

PCBs are available to purchase at a nominal AU$15 including international shipping for anyone who is interested in building their own shield. Shields can be purchased via the main website - https://diyflowbench.com

Please do not email me directly about purchasing - use the link on the website. Thanks.

For support with assembly, please visit the official support thread - [HERE](https://github.com/DeeEmm/DIY-Flow-Bench/discussions/372)

An interactive BOM is available [HERE](https://htmlpreview.github.io/?https://github.com/DeeEmm/DIY-Flow-Bench/blob/master/docs/iBom-DIY-FLOW-BENCH-V2-3.html)

Schematics are available below and can be downloaded from the PCB folder

![Schematic_DIY-FLOW-BENCH-V2-3](https://github.com/DeeEmm/DIY-Flow-Bench/blob/master/PCB/Schematic_DIY-FLOW-BENCH-V2-3.jpg?raw=true)
[Schematic_DIY-FLOW-BENCH-V2-3.pdf](https://github.com/DeeEmm/DIY-Flow-Bench/files/11220199/Schematic_DIY-FLOW-BENCH-V2-3.pdf)

------------------

### Known Issues

FLOW_VALVE_DIR_PIN (35) is assigned to a pin that can only be used as an input. A workaround is to jumper from the spare pin (4)
VCC_5V_PIN (2) is assigned to a pin that is also used by Wifi. A workaround is jumper to an unused pin

**Comment by LSRHokie** [comment]
Interested in picking up one of these shields from you if we can get it to the US .  Thanks again for all the hard work on this.

**Reply by DeeEmm** [reply]
Shipping to the US is not an issue. Will let you know when I received and validated them.

**Comment by DeeEmm** [comment]
Received the batch of new PCBs today.

![PXL_20240920_064345733](https://github.com/user-attachments/assets/7d050210-1593-46b7-911c-723528e2c93d)

![PXL_20240920_064331893](https://github.com/user-attachments/assets/55a52fb7-f000-4761-b535-806e70b0a3a2)


Initial impressions are good. Everything looks okay. Just need to transfer all of my components from the previous prototype and run through the validation tests.

Main differences from previous version are as follows:

![PXL_20240920_064402776](https://github.com/user-attachments/assets/91e8c311-ca38-48d7-af7b-c60085934fe5)

- Addition of a prototype area to allow for easy hacking / modification. This area includes a 3.3v and 0v bus as well as a grid of through holes to allow placement of components. This should hopefully cater for any custom applications. (inspiration taken from Speeduino Project. Credit to @noisymime aka Josh Stewart)

![PXL_20240920_064407415](https://github.com/user-attachments/assets/f265131a-1ecc-4d6e-918b-ce438dc58fb5)

- Updated pin layout for XGZ style pressure sensors, BME breakout boards and PCA level shifter BOB.

![PXL_20240920_064352798](https://github.com/user-attachments/assets/38dfc2f2-8d45-4701-b9bd-d340add4a16f)

- Corrected pin spacing for Buck style PSUs

**Reply by stefan63** [reply]
Nice, put me in the line of buyers of you not allready have :D

**Comment by DeeEmm** [comment]
I'm going to sit down today and work through getting one of these set up and tested. I've unfortunately caught Flu or maybe Covid from someone at the weekend so have been housebound this week. This has given me some time to work on the project. Only issue is that the headaches have made it difficult to see lol. Hopefully today I'll fare a bit better.

A note on BME280's. There is now an alternate to the BME280 in the form of the BME680. I was made aware of this by @KirikauKiwi who is using one on his M5 Stack Core 2 project #181. Seems that availability for these is now widespread, which is great for us.

The BME680 is available in several different footprints, including one that fits the existing footprint on the V2.3 shields.

![BME680](https://github.com/user-attachments/assets/386fdf43-e1f2-4152-8376-bd822d521063)

I've ordered one for testing as It is a good alternative to the BME280 which have flooded the market with fakes. I don't foresee any issues integrating it as communicates over I2C exactly the same as the BME280.

Will keep you posted.

**Reply by stefan63** [reply]
However it is a bit more expensive does not give higer precision and having functions not needed for this application.

**Reply by DeeEmm** [reply]
The price is comparable. BME280's appear to be cheaper on the whole as most of them are in fact BMP280s that are mis-sold. To guarantee that you get a genuine BME280 you need to pay upwards of $15. I've wasted ,money a few times on BME280's that turned out to be knock-off BMP's. Generic 680's are listed as low as $5 on ali-express if you wanted to take a punt

The Gas / VOC aspect of the 680 sensor does not need to be used and can just be ignored. Temp / Humidity and Baro is on par with the 280.

It is always good to have alternate component choices, not all components are available everywhere.  It's a similar story with the pressure and MAF sensors. The DIYFB code caters far many different types and is portable enough to cater for more.

We also have a user that is working through getting the code working on a M5-Stack Core2 with a BME680, so it seems like a good opportunity to include it in the code base.

**Comment by DeeEmm** [comment]
![PXL_20241030_101914380](https://github.com/user-attachments/assets/dd2cf2f8-9ae1-45aa-907c-e6d986d471ce)

Ok. Some good news. The initial assembly and first round of testing is a success.  The pin spacing on the buck PSU's is still a little bit out, but easy enough to bend the pins a little to fit.

Nice to that finally the BOB's for the level shifter and BME are now pin-compatible and so are no longer attached by jumper wires. This alone is gong to make my work desk a lot more tidy lol.

The board is now running with no issues so far. I still have some unit tests to undertake but expect to get through those tomorrow. I need to find where I stashed the Senoch XGZP pressure sensors. The footprints for these were incorrect on the previous PCB design. so keen to test and validate them. I know that a few people are either using these or plan to.

**Reply by stefan63** [reply]
Great news , good hard work done the last couple of weeks , Tnx!

**Reply by DeeEmm** [reply]
Cheers.

**Comment by DeeEmm** [comment]
I tested the Senoch XGZP pressure sensors today, which mount to the underside of the board and these now work.

Have to confess to frying one of my ESPs as I forgot to unplug it before setting the Buck PSU voltages. (ooops). A warning to anyone building a shield and something that I need to clarify in the Wiki. Fortunately I had a spare. But now I need to order another.

The only things that remain untested are now the SD card and Stepper drivers.

**Comment by DeeEmm** [comment]
Going to start taking a look at the SD card tonight. The code is already partially written, so hopefully just a case of commissioning it

**Comment by DeeEmm** [comment]
Well that didn't go to plan. :( 

```
[ 1332][E][sd_diskio.cpp:802] sdcard_mount(): f_mount failed: (3) The physical drive cannot work
```

Pretty consistent error. I've tried a few different cards, and a couple of different libraries, however it appears to be a common issue.

General consensus is that it is often resolved with a different SD card.  Which I will try next. So I guess this is on hold until then.

**Comment by DeeEmm** [comment]
Okay. I gave it one last stern stare down and it gave in...

```

DIY Flow Bench 
Version: V.2.0-RC.8 
Build: 24100101 
Initialising Hardware 
Scanning for I2C devices...
Found address: 72 (0x48)
Found address: 118 (0x76)
Found 2 device(s). 
Initialising ADS1115 
ADS1115 Initialised
Hardware Initialised 
Initialising Sensors 
Initialising BME280 
BME280 Initialised
Sensors Initialised 
File System Initialisation...
Complete.
Loading Configuration... 
Loading Lift Data... 
=== SPIFFS File system info === 
Total space:      1.26 MB 
Total space used: 68.88 KB 
=== SDCARD File system info === 
[   330][E][esp32-hal-cpu.c:110] addApbChangeCallback(): duplicate func=0x400f2938 arg=0x3ffbdee0
SD Card Type: SDSC
SD Card Size: 971MB
Listing directory: /
  DIR : .fseventsd
  DIR : .Spotlight-V100
  FILE: test.txt  SIZE: 1048576
  FILE: foo.txt  SIZE: 13
Creating Dir: /mydir
Dir created
Listing directory: /
  DIR : .fseventsd
  DIR : .Spotlight-V100
  FILE: test.txt  SIZE: 1048576
  FILE: foo.txt  SIZE: 13
  DIR : mydir
Removing Dir: /mydir
Dir removed
Listing directory: /
  DIR : .fseventsd
Listing directory: /.fseventsd
  FILE: fseventsd-uuid  SIZE: 36
  FILE: 0000000013d7b71b  SIZE: 57
  FILE: 0000000013d7b71c  SIZE: 72
  DIR : .Spotlight-V100
Listing directory: /.Spotlight-V100
  DIR : Store-V2
Listing directory: /.Spotlight-V100/Store-V2
  DIR : F25B30DD-AC70-4827-9117-9FC3DB5BB5FE
  FILE: VolumeConfiguration.plist  SIZE: 4516
  FILE: test.txt  SIZE: 1048576
  FILE: foo.txt  SIZE: 13
Writing file: /hello.txt
File written
Appending to file: /hello.txt
Message appended
Reading file: /hello.txt
Read from file: Hello World!
Deleting file: /foo.txt
File deleted
Renaming file /hello.txt to /foo.txt
File renamed
Reading file: /foo.txt
Read from file: Hello World!
1048576 bytes read for 4448 ms
1048576 bytes written for 5244 ms
Total space: 971MB
Used space: 2MB

Defaulting to AP Mode 
Creating WiFi Access Point:  DIYFB  
Access Point IP address: 192.168.4.1 
Multicast: http://diyfb.local 
Server Running 
```

I actually ran a 5V line to VCC on the SD card adaptor as it turns out it is a 5v adaptor and not a 3.3v adaptor. Guess that servers me right for not checking.

So. at least we now know that the SD card aspect of the PCB is all good and the expressif SD library looks like it is working too.

The actual SD card integration proper will want until V3 development. The current code is just a test.

Just one last test to do. (actually two). Stepper motors and the ULN2003 output, amd then we are ready to ship.

**Comment by DeeEmm** [comment]
OK so I have finally finished testing.

ULN works fine, valve adjust stepper is also okay, but there is an issue with the flow valve stepper direction pin (FLOW_VALVE_DIR_PIN). This pin has been assigned to a GPIO that can only be used for an input.

All is not lost however as a simple fix is to jumper to the spare output (4).

**Reply by stefan63** [reply]
Greate, that is what I call a minor issue with an easy solution.
You should be proud, not easy to get a PCB design right I know that by my limited experience.
I might post my ruff single sided PCB design in the future, however my board will not by far be that good looking :)

**Reply by DeeEmm** [reply]
Thanks.

I've done a few PCBs now across different projects and there is always issues, no matter how hard and thorough I try to be. I think this is just part of the process when the validator is the same person as the designer. It is very easy to overlook issues. "cannot see the wood for the trees" as the saying goes 😆 

Usually by the ninth or tenth iteration the bugs are ironed out 🤣

**Comment by DeeEmm** [comment]
@KirikauKiwi & @black-top can you flick me an email RE: PCBs - email addy is on my profile

**Reply by MinnesotaMike1** [reply]
Please, how do I order one of the PCB?

**Reply by DeeEmm** [reply]
HI. I haven't organised anything yet as I was waiting for the code testing to be completed. This should hopefully be done very soon.

**Comment by stefan63** [comment]
Me thinking loud again....
Maybe the BME should be on a lenth of wire so it can operate in free air without messuring temperature within or near the flow bench controll box? On my test setup I have a short 15cm twisted wire i guess a 1-2 meter wire would be possible with a low capacitance shielded 2 wire cable and a suitable PCB connector.

**Reply by DeeEmm** [reply]
There is a limitation on the distance that I2C will operate over. By design it is only supposed to be used between IC's on the same PCB but many use it as you describe. It may be worth experimenting to see how long the wires can be without experiencing I2C issues.

A low capacitance / oxygen free type cable might be the go. (twisted + shielded) These are commonly used to extend passive data transmission distances (RS232 etc). Would definitely recommend a proper shielded cable with drain to minimise potential interference.

**Reply by KirikauKiwi** [reply]
I will test this too, I had thought the same about the onboard BME280 and if it could be external.

[Spark fun](https://www.sparkfun.com/sparkfun-qwiicbus-endpoint.html) has a board that could be useful for this, it uses a chip to convert the I2C signal to run over two differential pairs and has an RJ45 connector to allow easy connection with an ethernet cable.

Another option could be the [LT3960](https://www.mouser.co.uk/datasheet/2/609/lt3960-1921786.pdf) it does a similar thing using the CAN bus physical layer, so it will work as a I2C Bridge. It is not an I2C CAN bus transceiver.

Third option could be add CAN bus to DIYFB and use an I2C CAN bus transceiver with the BME to put the I2C data on to the CAN network. This is what is shown in the [BANKS POWER](https://youtu.be/VUCo3REGrLE?t=670) video, their [Air Mouse](https://bankspower.com/products/sensor-airmouse-664200) is a BME sensor by the looks of it.

<img width="817" alt="Screenshot 2025-01-20 at 09 15 53" src="https://github.com/user-attachments/assets/eb9f02ef-0bbc-48b8-a1b9-e2a1b9fcb583" />

**Comment by stefan63** [comment]
Will do some tests and report back.

Den mån 20 jan. 2025 kl 06:40 skrev Mick Percy ***@***.***>:

**Comment by stefan63** [comment]
Some usefull reading on the I2C subject!
https://electronics.stackexchange.com/questions/461010/can-i-use-i2c-over-2m-cables

**Comment by ModchipMan** [comment]
Is there any issues I'm using V2.0 Board

**Reply by DeeEmm** [reply]
V2.0 board will still work fine.

The current version mainly addresses the pinout issues described in the V2.0 thread

**Comment by macturbo-eu** [comment]
Hi guys. Im very insterested in this complete pcb board ready to work, if you want to make 1-2pcs complete for me i can send the money in advance and wait till its done. Also sent an email, im not good with electronics. 
Want to modify my turbo vnt flow bench using bosch maf ...123. Maximum readout is under 300kg of air per hour.usually 30-90kg max flow thru turbo.

**Reply by DeeEmm** [reply]
The PCBs are DIY. 

There are plans to sell completed PCBs but they are not finalised yet. 

What is your timeframe?

**Reply by macturbo-eu** [reply]
as fast as possible lol but ill wait as long as its needed, hopefully not months. its something i really want to do in order to improve my work flow in the shop. if you reply on email send me your data and ill book it. 
otherwise i can talk to my electronics guy and see if he wants to do it if you think its gonna take you way too long.

**Reply by DeeEmm** [reply]
We are just at data validation stage. This is the only thing left to do.

**Comment by MinnesotaMike1** [comment]
I’d like to build my own but I’m not sure where to begin. I’ve read so much
info in the last three weeks and now I can’t remember where exactly I read
what.

**Reply by DeeEmm** [reply]
The Shield design section of the WIKI is a good starting point

**Comment by MinnesotaMike1** [comment]
If I purchase this board will it be the correct one to use for the DIYFB?
I have a differential pressure orifice pitot style bench and I want to
learn how to add the DIYFB electronics so I don't have to do the
calculations by hand.  Thanks

https://a.co/d/e5CbNAy

**Reply by DeeEmm** [reply]
No. That is is not an ESP32. It is an ESP8266 which was the ESP32's predecessor. The code will not run on that as it has a different processor and memory.

The official board is the ESP32 Wemos R32 D1.

https://www.makerstore.com.au/product/elec-esp32-d1-r32

Wemos was the original manufacturer but there are now various clones available. It pays to shop around but generally the cheapest place to find them is aliexpress...

https://www.aliexpress.com/item/1005006880266536.html

Amazon tends to be a fair bit more expensive...

https://www.amazon.com.au/dp/B07WFZCBH8

**Comment by MinnesotaMike1** [comment]
Awesome, thanks for the info!  Is there someplace I can order the blank pcb
shield board from?

**Reply by DeeEmm** [reply]
The PCB are still not released. I'm just holding off until validation is complete. Shouldn't be too much longer. A few weeks at most hopefully.

**Comment by MinnesotaMike1** [comment]
Ok, thanks a lot. I apologize for all the novice questions. This will be a
fun project for me. Lots of new things to learn.

**Reply by DeeEmm** [reply]
All good. This is why we are doing this. To help everyone and anyone who wants to have ago themselves. 👍

**Comment by macturbo-eu** [comment]
i will wait for a full board with all the gadgets on it. if you need to
order parts i can send the money in advance (ppal gift).

the VNT flow benches on the market blow 0.3bar constant thru the turbos and
measure flow. i see there is a part dedicated on this board for flow
control. nice!

On Fri, 14 Feb 2025 at 09:45, Mick Percy ***@***.***> wrote:

> The PCB are still not released. I'm just holding off until validation is
> complete. Shouldn't be too much longer. A few weeks at most hopefully.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DeeEmm/DIY-Flow-Bench/discussions/171#discussioncomment-12197254>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPNUBI2UD46B2BJ356K2DHT2PWNIBAVCNFSM6AAAAABOASG4WCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEMJZG4ZDKNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.macturbo.eu
https://www.facebook.com/cp3upgrade
Mac Performance Tuning
+40735886208

**Reply by DeeEmm** [reply]
So do the VNT setups blow through the exhaust turbine and then the flow is measured from the compressor side?

We did have an enquiry early on about using the project to measure turbo's but they only asked about the possibility of adding a trigger to the compressor wheel to measure turbine rpms. I added it to the things-to-do list but we have not got to that stage yet.

Would be good to have someone using DIYFB for this kind of application so that we can develop the code to cater for it. It is not a field that I have delved into myself, although I've run turbo cars in the past. I will start a dedicated discussion for this as I would like to learn more

**Comment by MinnesotaMike1** [comment]
I'm so excited about this project!  You guys have done an awesome job putting the whole project together.  I will order a blank PCB to the US as soon as they are ready.  I'm looking forward to putting it together myself.  I ordered a ESP32 last night and am very excited to get started with it.  Thanks for putting together such an amazing flow bench resource, this is really outstanding work.

**Reply by DeeEmm** [reply]
Thanks Mike. Its cool to see so much interest.

**Comment by macturbo-eu** [comment]
How it works is:
-there is a calibration plate with hole and flow is measured at 0.3bar
constant pressure thru the system.
-mount the turbo on exhaust side, shaft is blocked so its not spinning,
close VNT at 80%
-start measuring , engine pushes constant 0.3bar thru VNT exhaust side and
flow is measured on the exhaust side. Adjustments are made to reach desired
flow by opening or closing the vnt setting.
This way we know when turbo is on the car it will work correctly and make
boost as it should.

Some benches work with pressure 0.3bar some with vacuum. Vaccum would be
easier to operate as you suck it thru exit side of turbo. Pushing sometimes
is more difficult because of the shape of turbo manifold..have to block all
leaks or holes.

On Sat, 15 Feb 2025 at 05:18, Mick Percy ***@***.***> wrote:

> So do the VNT setups blow through the exhaust turbine and then the flow is
> measured from the compressor side?
>
> We did have an enquiry early on about using the project to measure turbo's
> but they only asked about the possibility of adding a trigger to the
> compressor wheel to measure turbine rpms. I added it to the things-to-do
> list but we have not got to that stage yet.
>
> Would be good to have someone using DIYFB for this kind of application so
> that we can develop the code to cater for it. It is not a field that I have
> delved into myself, although I've run turbo cars in the past. I will start
> a dedicated discussion for this as I would like to learn more
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DeeEmm/DIY-Flow-Bench/discussions/171#discussioncomment-12207124>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPNUBI7US2UZ55DQWX3E4D32P2WWTAVCNFSM6AAAAABOASG4WCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEMRQG4YTENA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.macturbo.eu
https://www.facebook.com/cp3upgrade
Mac Performance Tuning
+40735886208

**Reply by DeeEmm** [reply]
Thanks for that explanation.

What determines the correct, or 'best' VNT settings? Are you just looking for optimal bypass flow? Or a specific flow value?

**Comment by macturbo-eu** [comment]
New turbos were measured by flow bench producers so there is a database.
They use new turbos to measure.

**Reply by macturbo-eu** [reply]
What do you think its the time frame for these boards to be ready? Any estimate?

**Comment by DeeEmm** [comment]
Ok. Validation has been a lot slower than I would have liked and there is still not enough submitted data to make any kind of informed decision about MAF data validity. So rather than wait any longer I have decided to release the shields for purchase on the proviso and understanding that until validation has been completed, MAF data may have inaccuracies. The goal here is just to get more benches built and hopefully as a result more people contributing. With every additional piece of data submitted, we will improve the overall accuracy of the system. The more data that is submitted, the more accurate it will become.

In an ideal world I would have preferred to release the board only when everything is 100% validated, but as in everything there has to be compromise and things appear to have stalled without this additional input.

So if you have been waiting to purchase a board, you can now submit an order via the main website.

Cost is AU$15 which includes free international shipping. This basically covers the costs of the board manufacture along with shipping and handling. 

I've shipped PCBs to those contributors that I previously promised. (sorry for the delay)

Lets get this ball rolling.

/DM

**Reply by MinnesotaMike1** [reply]
I filled out the order form but I didn't hear back about payment so I'm worried I didn't fill out the order correctly.  I just sent in another order.  Thanks for making all of this available.  I really admire your work!

**Reply by DeeEmm** [reply]
Sorry Mike, The auto responder was not set up properly. You should now have received an email.

**Comment by macturbo-eu** [comment]
Hi, im in. Can you make me a complete board with all the parts on it? i pay and wait, not an issue, just let me know how much to send. ill start building the VNT flow bench. 
I would like to get this started. 
Also i have access to 2 vnt flow benches to get data and compare against the maf we are gonna use, compare flow from one bench to the other.

**Reply by DeeEmm** [reply]
Assembled boards is something that I plan to offer but am not set up for at this point in time. Let me look into it for you.

**Reply by macturbo-eu** [reply]
how is it looking ?

**Reply by macturbo-eu** [reply]
question: do you position all the parts by hand and solder them one by one? maybe its something i could do myself. im quite handy with this stuff even if it takes time i dont have

**Reply by DeeEmm** [reply]
Sorry I have not had a chance to look at this yet. Unfortunately I am very time-poor.

With regards to assembling it yourself. The assembly is pretty straightforwards as it is designed to use all through-hole parts. So anyone with a soldering iron and some patience can assemble this. 

The power supply bucks, BME280, ADS and level shifters are already mounted to BOBs (BreakOut Boards), so you only need to solder pins or sockets to the PCB and plug them in. Again this design is specifically to make it as easy as possible for a novice to assemble.

Generally when assembling the PCB, you can solder all the resistors, and then cut back the leads, before moving on to the capacitors, and then the pin-sockets and finally the pressure sensors. It should take not more than an hour or two max.

My plan for assembled boards is to migrate the design to use SMD components so that I could order them in pre-soldered, as I just don't have the time to do them myself. But getting the time to migrate the design to SMD is an amount of work in itself, so is not something that will happen quickly. 

If you are in a rush, I would definitely suggest that you consider having a go yourself. It's not too difficult.

**Comment by macturbo-eu** [comment]
yes please, i can send the money in advance so you can order everything

On Fri, 14 Mar 2025 at 12:33, Mick Percy ***@***.***> wrote:

> Assembled boards is something that I plan to offer but am not set up for
> at this point in time. Let me look into it for you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DeeEmm/DIY-Flow-Bench/discussions/171#discussioncomment-12499356>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BPNUBI6IYVJMZYQVFIEIH4D2UKWAPAVCNFSM6AAAAABOASG4WCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTENBZHEZTKNQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.macturbo.eu
https://www.facebook.com/cp3upgrade
Mac Performance Tuning
+40735886208

**Comment by macturbo-eu** [comment]
I will give it a try myself . Can somebody make a list with the components
to be ordered. I want to assemble a full board with all the options.

**Reply by DeeEmm** [reply]
There is a full BOM in the WIKI.

It is advisable to read through the WIKI before ordering anything as there are several different sensor configurations.

**Reply by macturbo-eu** [reply]
Ok, let me know when you are ready to ship. i would like to start

**Reply by DeeEmm** [reply]
Bare PCBs can be ordered via the website - https://diyflowbench.com/

**Comment by DeeEmm** [comment]
I'm going to close this discussion out as the design for V2.3 is essentially complete

In it's place I've created a support thread for V2.3 PCB assembly - [HERE](https://github.com/DeeEmm/DIY-Flow-Bench/discussions/372)

This is to provide support for anyone who has purchased the V2.3 PCB.
